pynautobot/core/endpoint.py

```diff
@@ -690,3 +690,39 @@ def run(self, *args, api_version=None, **kwargs):
         ).post(args[0] if args else kwargs)
 
         return response_loader(req, self.return_obj, self)
+
+
+class GraphqlEndpoint(Endpoint):
```

Comment:
Do we need to add this new endpoint here: https://github.com/nautobot/pynautobot/blob/develop/pynautobot/core/app.py#L63-L66

Reply:
Not in my testing, but not sure of everything added for the `jobs` integration.  Is it possible that `jobs` can be a child object of something else?  I don't think the same is true for a saved query.

Reply:
It looks like it's required to be able to call the run method directly with `nb.extras.graphql_queries.run()`:

```python
In [11]: nb.extras.jobs.run
Out[11]: <bound method JobsEndpoint.run of <pynautobot.core.endpoint.JobsEndpoint object at 0x7f9b38d05d00>>

In [12]: nb.extras.graphql_queries.run
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-bc5064448ad1> in <cell line: 1>()
----> 1 nb.extras.graphql_queries.run

AttributeError: 'Endpoint' object has no attribute 'run'
```

Does it make sense to call that directly like that? I'm not sure if that's necessarily a requirement, but maybe we should add it for consistency if it does.

Reply:
Agreed, added it for consistency.